src/shared/analytics/analytics.background.ts

```diff
@@ -86,24 +114,20 @@ function trackAppEvents({ account }: { account: Account }) {
     mixpanelTrack(account, 'DApp: DApp Connection', mixpanelParams);
   });
 
-  emitter.on('screenView', async (data) => {
-    const params = createParams({
-      request_name: 'screen_view',
-      wallet_address: data.address,
-      screen_name: data.pathname,
-      previous_screen_name: data.previous,
-      screen_size: data.screenSize,
-      window_type: data.windowType,
-    });
-    sendToMetabase('screen_view', params);
-    const portfolio = await getOwnedWalletsPortolio(account);
-    const mixpanelParams: Record<string, unknown> = {
-      ...omit(params, ['request_name', 'wallet_address']),
-      total_balance: portfolio?.total_value ?? 0,
-      ...getChainBreakdown(portfolio),
-    };
-    mixpanelTrack(account, 'General: Screen Viewed', mixpanelParams);
-  });
+  emitter.on('screenView', (data) =>
+    trackScreenView({
```

Comment:
There was a request from analytics:
`Please delete property [blockchain_id](id from backend)_balance from other events, except the previous one`
So we should remove chain balances from screen view events, and keep them only in app opened event

Reply:
ah, thanks, nice catch, formal-natural language parsing issue